modules/standard/Math.chpl

```diff
@@ -276,11 +369,29 @@ module Math {
   }
 
 
-  /* Returns the complex conjugate of the argument `a`.
+  /* Returns the complex conjugate of the argument `z`.
    
-     :rtype: The type of `a`.
+     :rtype: The type of `z`.
   */
-  inline proc conjg(a: complex(?w)) : complex(w) return (a.re, -a.im):complex(w);
+  inline proc conjg(z: complex(?w)): complex(w) {
+    extern proc conjf(z: complex(64)): complex(64);
+    extern proc conj(z: complex(128)): complex(128);
+    if w == 64 then
+      return conjf(z);
+    else
+      return conj(z);
+  }
+
+
+  /* Returns the projection of `z` on a Riemann sphere. */
+  inline proc proj(z: complex(?w)): real(w/2) {
+    extern proc cprojf(z: complex(64)): real(32);
+    extern proc cproj(z: complex(128)): real(64);
+    if w == 64 then
+      return cprojf(z);
+    else
+      return cproj(z);
+  }
```

Comment:
I'd consider naming this one cproj for similar reasons to arg; it only applies to complex and has a potentially likely to conflict name.